### DISCUSSION TITLE
Is there a workaround for multiple values for the same attributes?

### DISCUSSION THREAD

**Discussion by kes2464** [open-telemetry/opentelemetry-java-instrumentation#10440] [Q&A] [answered]
One of applications I have has 2 servlets in one JVM, which have their own Hikari connection pool.

```
[PeriodicMetricReader-1] WARN io.opentelemetry.sdk.metrics.internal.state.AsynchronousMetricStorage - Instrument db.client.connections.max has recorded multiple values for the same attributes: {pool.name="test-pool"}
```

However, Opentelemetry instrumenter does not distinguish them, so either one is being dropped.
Is there a way to add a label, or an attribute from a servlet name?

**Comment by laurit** [comment, marked as answer]
Have you considered giving these pools different names?

**Reply by kes2464** [reply]
It isn't a simple change for me due to how it's set in the code, but if there's no other way, then that should be the way to go.

**Reply by laurit** [reply]
As an alternative you can disable the instrumentation that produces these metrics with `-Dotel.instrumentation.hikaricp.enabled=false`
